experimental/dds/tree2/src/test/edit-manager/editManager.spec.ts

```diff
@@ -636,10 +646,12 @@ function runUnitTestScenario(
 						seq,
 						...localIntentions,
 					];
-					assert.deepEqual(
-						manager.addSequencedChange(commit, commit.seqNumber, commit.refNumber),
-						asDelta(expected),
+					const actual = manager.addSequencedChange(
```

Comment:
I'm assuming these are left over from debugging but it seems nice to keep them pulled out. Tiny nit - name this one `delta` to be consistent with the one in the ACK case